source/eventcore/drivers/posix/sockets.d

```diff
@@ -927,6 +930,13 @@ private void setSocketNonBlocking(SocketFD sockfd)
 	}
 }
 
+private void setSocketCloseOnExec(SocketFD sockfd)
+@nogc nothrow {
+	version (Windows) {} else {
+		() @trusted { fcntl(cast(int)sockfd, F_SETFL, O_CLOEXEC, 1); } ();
```

Comment:
Ooops...sorry, O_CLOEXEC is defined only FreeBSD in druntime.

Reply:
fixed https://github.com/vibe-d/eventcore/pull/38/commits/2f3f7b1773f88539d0dc3ed8dce1fcecaa950d27